extension/wrappers.h

```diff
@@ -315,6 +367,78 @@ class CBasePlayer :
 
 		pPlayerInfo->ChangeTeam(teamIndex);
 	}
+
+#if SOURCE_ENGINE == SE_LEFT4DEAD2
+	CBasePlayer* GetSplitScreenPlayerOwner()
```

Comment:
`IVEngineServer` has these:
```
bool IsSplitScreenPlayer( int ent_num )
edict_t *GetSplitScreenPlayerAttachToEdict( int ent_num )
edict_t *GetSplitScreenPlayerForEdict( int ent_num, int nSlot )
```

Use these instead of bulking up more gamedata.